src/GRA.Domain.Service/VendorCodeService.cs

```diff
@@ -206,15 +212,17 @@ var jobDetails
                 {
                     using (var stream = new FileStream(fullPath, FileMode.Open))
                     {
+                        int branchColumnId = 0;
                         int couponColumnId = 0;
+                        int detailsColumnId = 0;
                         int orderDateColumnId = 0;
                         int shipDateColumnId = 0;
-                        int detailsColumnId = 0;
                         var issues = new List<string>();
                         int row = 0;
                         int totalRows = 0;
                         int updated = 0;
                         int alreadyCurrent = 0;
+                        VendorCodeType vendorCodeType = null;
```

Comment:
This value doesn't seem to ever be set?

Reply:
Yeah, this was working but then I moved out functionality into a separate method. I've fixed it so it uses this now - it caches the VendorCodeType until the next loop saving a lookup if it's the same VendorCodeType (spoiler alert: it always is in our case).